pkg/console/util.go

```diff
@@ -447,6 +447,25 @@ func saveTemp(obj interface{}, prefix string) (string, error) {
 	return tempFile.Name(), nil
 }
 
+func roleSetup(c *config.HarvesterConfig) {
+	if c.Role == "" {
+		return
+	}
+	if c.Labels == nil {
+		c.Labels = make(map[string]string)
+	}
+	switch c.Role {
+	case config.RoleMgmt:
+		c.Labels[util.HarvesterMgmtNodeLabelKey] = "true"
+	case config.RoleWorker:
+		c.Labels[util.HarvesterWorkerNodeLabelKey] = "true"
+	case config.RoleWitness:
+		c.Labels[util.HarvesterWitnessNodeLabelKey] = "true"
+	default:
+		logrus.Warnf("unknown role %s", c.Role)
```

Comment:
Can we add a check in validato.go if an inputted role is unsupported.

Reply:
Updated!
Now supported role: `default`, `management`, `witness` and `worker`

If the user does not set up a role, it will be default.